pkg/util/kubernetes/kubelet/kubelet_client.go

```diff
@@ -205,8 +206,8 @@ func getKubeletClient(ctx context.Context) (*kubeletClient, error) {
 
 	// Checking HTTPS first if port available
 	var httpsErr error
-	if kubeletHTTPSPort > 0 {
-		httpsErr = checkKubeletConnection(ctx, "https", kubeletHTTPSPort, kubeletPathPrefix, potentialHosts, &clientConfig)
+	if kubeletHTTPSPort > 0 && kubeletHTTPSPort <= math.MaxUint16 {
```

Comment:
Is this new check needed ?
As the value now comes from `strconv.ParseUint(…, 10, 16)`, it can never be bigger than `math.MaxUint16`.